vector/src/main/res/layout/fragment_timeline.xml

```diff
@@ -87,8 +86,19 @@
         app:closeIcon="@drawable/ic_close_24dp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/removeJitsiWidgetView"
-        tools:visibility="visible" />
+        app:layout_constraintTop_toBottomOf="@id/removeJitsiWidgetView"/>
+
+    <im.vector.app.core.ui.views.TypingMessageView
+        android:id="@+id/typingMessageView"
+        app:layout_constraintBottom_toTopOf="@id/composerLayout"
+        app:layout_constraintTop_toBottomOf="@id/timelineRecyclerView"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        android:layout_width="0dp"
+        android:paddingStart="20dp"
+        android:paddingEnd="20dp"
+        android:visibility="invisible"
```

Comment:
Maybe make it visible in the layout editor

Reply:
Done. 
Thanks !